lib/tasks/email_giftcards.rake

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+namespace :email_giftcards do
+  desc 'Email customers the gift cards that they ordered'
+  task :send_emails => :environment do
+    desc 'send emails'
+
+    unique_recipients = GiftCardDetail
+                        .joins(:item, :recipient)
+                        .where(items: { refunded: false })
+                        .distinct.pluck(:recipient_id)
+
+    unique_recipients.each do |id|
+      html = '<!DOCTYPE html>' \
+              '<html>' \
+              '<head>' \
+              "  <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />" \
+              '</head>' \
+              '<body>' \
+              '<p> Thank you for supporting local chinatown businesses! As our merchants open up, you will be able to redeem the giftcards you have purchased below.</p>'
```

Comment:
Chinatown is a properly noun

Reply:
I'd consider making it simpler and just removing this sentence `As our merchants open up, you will be able to redeem the giftcards you have purchased below.`

Reply:
Thank you for supporting local Chinatown businesses! Your gift card to <merchant name> can be found here: <link to gift card>

Current balance: $XX.XX

Reply:
But I'd just swing it by @nanxiy and see what she thinks

Reply:
> Chinatown is a properly noun

I like how you called out his grammar with your own grammatical mistake 😂. But in all seriousness good catch